private/src/avb_watchdog/IasWatchdogInterface.cpp

```diff
@@ -46,23 +46,15 @@ IasWatchdogInterface::~IasWatchdogInterface()
 IasWatchdogResult IasWatchdogInterface::registerWatchdog(uint32_t timeout, std::string watchdogName)
 {
   IasWatchdogResult result = IasMediaTransportAvb::IasResult::cOk;
+  int signed_timeout = timeout;
```

Comment:
Ditto. Where is it possible for someone to send a negative value here? The API is quite clear about `uint32_t`.